src/pre_defined_classes/mathematical_program/hard/q2/JavademicTest.java

```diff
@@ -0,0 +1,20 @@
+package pre_defined_classes.mathematical_program.hard.q2;
+
+import global.BaseTest;
+import global.variables.*;
+
+public class JavademicTest extends BaseTest {
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("It will take "),
+                new IntegerLiteral(40),
+                new StringLiteral(" days to infect "),
+                new IntegerLiteral(1000000),
```

Comment:
`DoubleLiteral`

Reply:
Why double literal?

Reply:
I think this is because some of the distractors return doubles

Reply:
The only distractor that would change the output from an integer is double t = Math.exp( xt / x0 ) / Math.exp( 1 + r ); which actually causes the program to print "Infinity" so, is the recommendation that I change it to String instead?